src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
```

Comment:
move font styles to html tag
also, body doesn't have initial paddings, remove it